packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -50,6 +50,18 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
         @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService
     ) { }
 
+    registerTermCommands(termWidget: TerminalWidget, commands: CommandRegistry) {
+        const CLEAR_OUTPUT: Command = {
+            id: 'terminal:' + termWidget.id + ':clearOutput',
+            label: 'Clear Terminal Output'
```

Comment:
`Terminal: Clear`

the general conventional is: `category: command` and `...` if a command requires any additional args, `Open New Terminal` is an exception

Reply:
Changed label